solr/solrj/src/java/org/apache/solr/common/cloud/ZkMaintenanceUtils.java

```diff
@@ -33,8 +33,8 @@
 import java.util.function.Predicate;
 import java.util.regex.Pattern;
 
-import org.apache.commons.lang.StringUtils;
 import org.apache.solr.client.solrj.SolrServerException;
+import org.apache.solr.common.StringUtils;
```

Comment:
Was the removal of the commons-lang 2.6 dependency tied _only_ to this file or was the older ZooKeeper using it?  If it was only this file, the PR should have been split.  And FYI Solr depends on Commons-Lang-3 so StringUtils is there as well and has these methods.